mtp_cashbook/apps/mtp_auth/models.py

```diff
@@ -19,6 +19,28 @@ def save(self, *args, **kwargs):
     def is_authenticated(self, *args, **kwargs):
         return True
 
+    @property
+    def username(self):
+        return self.user_data.get('username')
+
+    @property
+    def get_full_name(self):
+        if not hasattr(self, '_full_name'):
+            name_parts = [
+                self.user_data.get('first_name'),
+                self.user_data.get('last_name')
+            ]
+            self._full_name = ' '.join(filter(None, name_parts))
+        return self._full_name
+
+    @property
+    def prison(self):
+        try:
+            return self.user_data.get('prisons', [])[0]
```

Comment:
Would it be better to return the entire array of prisons the user has access to here, enabling something like `if prison in user.prisons`? That'd also remove the need for the `[0]`, I think?